### ISSUE TITLE
Wait time not long enough between questions.

### ISSUE THREAD

**Issue by mitchwillman** [goldsziggy/alexa_drinkmaster#1]
If you do not say next pretty much immediately, it will end the game.

**Comment by goldsziggy** [comment]
Hmmm, last I checked I didn't think Amazon allowed us to customize the wait time (I will check on this though!).

I did add the ability for the game to 'pick-up' where you left off, have you seen that not work?

**Comment by mitchwillman** [comment]
Yes, anytime we waited more than, say 10 seconds to say next it would end
the game and return to the main menu.

**Comment by wilsonge** [comment]
:+1: on this. I have exactly the same issue - it returns back to the main menu after like 10 seconds and prompts you to restart the game
